main.py

```diff
@@ -1065,6 +1065,12 @@ def run_analysis(self) -> None:
                                                        )
                 raise Exception(f"An Unexpected error has occurred running the {self.scan_mode} scan")
 
+            # Add the scanned urls to the report
+            scannedUrls = open('./spidered_urls.txt', 'r').read().splitlines()
+            data = json.load(open(Constants.REPORT_SCAN_RESULT_FILE, 'r'))
+            data['scannedUrls'] = scannedUrls
```

Comment:
spideredUrls (since we use scannedUrl in the BE)

Reply:
as approved in te be it will be discoveredUrls